core/CORE_NAILS_Exceptions.php

```diff
@@ -40,7 +40,12 @@ public function show_error($heading, $message, $template = 'error_general', $sta
         } else {
 
             $sTemplate = APPPATH.'errors/' . $template . '.php';
-            if (!file_exists($sTemplate)) {
+
+            if (!class_exists('CI')) {
+
+                echo implode("\n", array_filter(['ERROR:', $heading, $message]));
```

Comment:
I think we should use the same mechanic as below (i.e `_NAILS_ERROR`). Unless there was a reason you chose to do it like this? Probably even combine into a single conditional.

```
if (!class_exists('CI') || !file_exists($sTemplate)) 
```

Reply:
@hellopablo two reasons why I'm not doing this:
- `_NAILS_ERROR` is defined in `index.php`, which does not get loaded during a CLI runtime
- It formats errors in a load of HTML markup which is hard to read on the CLI

Reply:
Clever man. 🌮